### PR TITLE
Branch 12 -- add descriptions of question 2

### PR DESCRIPTION
**Pull request by sw3614**
For this pull request, I improved the coding comment by adding the description of how to convert a dataset to a wide format what functions are used. The reason for doing this is that I want to let viewers understand the method of converting to a wide format better. I perceive it as a useful tool for viewers and graders to know how the author is thinking about this question.

### CODE REVIEW

**Comment by sw3614** [comment]
[3b70927](https://github.com/QMSS-GR5069-Spring-2022/gr5069-homework-2-JeremyHBUT/commit/3b70927a47d0ef0b1fe5f34da2a5890e2d72fe73)
Final Submission of homework 2

**Comment by JeremyHBUT** [comment]
Accurate

File: code_improvement/MDS_homework2_original.ipynb [hunk 1]
```diff
@@ -0,0 +1,2055 @@
...
+      "text/plain": [
+       "     YearStart LocationAbbr  LocationDesc Question DataValue Stratification1\n",
+       "340       2015           AL       Alabama  Poverty      18.5         Overall\n",
+       "369       2014           TX         Texas  Poverty      17.2         Overall\n",
+       "379       2011           NM    New Mexico  Poverty      21.5         Overall\n",
+       "473       2011           WY       Wyoming  Poverty      11.3         Overall\n",
+       "489       2011           ND  North Dakota  Poverty      12.2         Overall"
+      ]
+     },
+     "execution_count": 25,
```
Comment (JeremyHBUT) [inline code_improvement/MDS_homework2_original.ipynb:904]: impressive

**Review by JeremyHBUT — COMMENTED** [review]
accurate data

File: code_improvement/code_change2/mds_homework2_code_change2.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+LargestInc5 = AvgGrowth.sort_values('AverageGrowth',ascending = False).head(5)
+LargestInc5
+
+
+# In[71]:
+
+
+LargestDec5 = AvgGrowth.sort_values('AverageGrowth').head(5)
+LargestDec5
+
```
Comment (JeremyHBUT) [inline code_improvement/code_change2/mds_homework2_code_change2.py:170]: Excellent description

**Pull request merged**